.circleci/config.yml

```diff
@@ -105,12 +105,15 @@ jobs:
             python3.9 -m pip install wheel
             python3.9 -m pip install -r requirements.txt
             sudo ln -sf /usr/bin/python3.9 /usr/bin/python
+
       - run:
-          name: Run
+          name: Run Parametric tests
           environment:
             TEST_LIBRARY: cpp
             DD_TRACE_CPP_COMMIT: << pipeline.git.revision >>
-          command: cd system-tests && ./build.sh -i runner && ./run.sh PARAMETRIC --log-cli-level=DEBUG 
+          command: |
+            echo "https://github.com/DataDog/dd-trace-cpp@$DD_TRACE_CPP_COMMIT" > ~/project/system-tests/binaries/cpp-load-from-git
```

Comment:
Does this actually work?

Reply:
If it will work once we merge the PR: https://github.com/DataDog/system-tests/pull/1983

Reply:
Could you just set the version of `dd-trace-cpp` instead of the whole github url?

Reply:
It is to maintain a certain similarity with other tracers. ie: https://github.com/DataDog/system-tests/blob/main/utils/scripts/load-binary.sh#L230